docs/mkdocs/docs/index.md

```diff
@@ -1,387 +1,495 @@
 
-#### S3 configuration
+### Library Setup
 
-There are two methods to configure S3 access. If you happen to know the access and secret key, simply connect as follows:
+ArcticDB is geared towards storing many (potentially millions) of tables. Individual tables (DataFrames) are called _symbols_ and 
+are stored in collections called _libraries_. A single _library_ can store many symbols.
 
+_Libraries_ must first be initialized prior to use:
+
+```python
+ac.create_library('data')  # fixed schema - see note below
+ac.list_libraries()
+```
+output
 ```python
->>> import arcticdb as adb
->>> ac = adb.Arctic('s3://ENDPOINT:BUCKET?region=blah&access=ABCD&secret=DCBA')
+['data']
 ```
 
-Otherwise, you can delegate authentication to the AWS SDK (obeys standard [AWS configuration options](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html)):
+The library must then be instantiated in the code ready to read/write data:
 
 ```python
->>> ac = adb.Arctic('s3://ENDPOINT:BUCKET?aws_auth=true')
+library = ac['data']
 ```
 
-Same as above, but using HTTPS:
+Sometimes it is more convenient to combine library creation and instantiation using this form, which will automatically create the library if needed, to save you checking if it exists already:
 
 ```python
->>> ac = adb.Arctic('s3s://ENDPOINT:BUCKET?aws_auth=true')
+library = ac.get_library('data', create_if_missing=True)
 ```
 
-!!! s3 vs s3s
+!!! info "ArcticDB Static & Dynamic Schemas"
 
-    Use `s3s` if your S3 endpoint used HTTPS
+    ArcticDB does not need data schemas, unlike many other databases. You can write any DataFrame and read it back later. If the shape of the data is changed and then written again, it will all just work. Nice and simple.
 
-##### Connecting to a defined storage endpoint
+    The one exception where schemas are needed is in the case of functions that modify existing symbols: `update` and `append`. When modifying a symbol, the new data must have the same schema as the existing data. The schema here means the index type and the name, order, and type of each column in the DataFrame. In other words when you are appending new rows they must look like the existing rows. This is the default option and is called `static schema`.
 
-Connect to local storage (not AWS - HTTP endpoint of s3.local) with a pre-defined access and storage key:
+    However, if you need to add, remove or change the type of columns via `update` or `append`, then you can do that. You simply need to create the library with the `dynamic_schema` option set. See the `library_options` parameter of the ([`create_library`](api/arctic/#arcticdb.Arctic.create_library)) method.
+
+    So you have the best of both worlds - you can either enforce a static schema on your data so it cannot change by modifying operations, or allow it to be flexible.
```

Comment:
I find this misleading as if you can change from static->dynamic (even though the next line explains that it's set at creation time)